### PR TITLE
Create file4.txt

### PR DESCRIPTION
**Pull request by Andy-githubtest**
Create file number 4. Please review.

### CODE REVIEW

File: file4.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Another file for a PR
```
Comment (bob2-githubtest) [inline file4.txt]: Nice code. I like it.

Comment (bob2-githubtest) [inline file4.txt]: I'd like it formatted better.

**Review by bob2-githubtest — CHANGES_REQUESTED** [review]

File: file4.txt (hunk 1 above)
Comment (maksimkviatkouskicdemo) [reply file4.txt]: Me too!

Comment (maksimkviatkouskicdemo) [reply file4.txt]: Agree.

**Review by maksimkviatkouskicdemo — CHANGES_REQUESTED** [review]

**Review by maksimkviatkouskicdemo — CHANGES_REQUESTED** [review]
I want some more changes.

**Review by bob2-githubtest — APPROVED** [review]
Lookin'good!

**Review by maksimkviatkouskicdemo — APPROVED** [review]
On a second thought I'd like to approve it.

**Comment by bob2-githubtest** [comment]
It's a general discussion comment from Bob.

**Pull request merged**